lib/nexpose/api_request.rb

```diff
@@ -97,27 +97,31 @@ def execute(options = {})
             end
           end
         end
-        # This is a hack to handle corner cases where a heavily loaded Nexpose instance
-        # drops our HTTP connection before processing. We try 5 times to establish a
-        # connection in these situations. The actual exception occurs in the Ruby
-        # http library, which is why we use such generic error classes.
-      rescue OpenSSL::SSL::SSLError => e
```

Comment:
I know code inspection complains about these but capturing the object makes runtime debugging easier, I would prefer they be kept.